packages/docs/src/pages/startup/browser-support.md

```diff
@@ -3,6 +3,6 @@ title: Browser support
 weight: 5
 ---
 
-The CMS design system follows the [2% rule](https://gds.blog.gov.uk/2012/01/25/support-for-browsers/): we officially support any browser above 2% usage and the last 2 browsers versions for each browser as observed by [analytics.usa.gov](https://analytics.usa.gov/). **Currently, this means that the newest versions of Chrome, Firefox, Safari, Internet Explorer 11, Edge, and Samsung Internet are supported.**
+The CMS design system follows the [2% rule](https://gds.blog.gov.uk/2012/01/25/support-for-browsers/): we officially support any browser above 2% usage and the last 2 browsers versions for each browser as observed by [analytics.usa.gov](https://analytics.usa.gov/). For specific browser details check out [browserl.ist](https://browserl.ist/?q=last+2+versions%2C+%3E2%25). **Currently, the latest 2 versions of Chrome, Firefox, Safari, Internet Explorer 11, and Edge, and Samsung Internet are supported.**
```

Comment:
Looks like Samsung Internet is still here. And super small nitpick, but what do you think about changing the link from `browserl.ist` to `browserlist`?